packages/rest-client/CHANGELOG.md

```diff
@@ -1,8 +1,12 @@
 # Change log
 
+## 0.0.1-alpha.9
+
+Improve javadoc for `RestClient` to document the throwing of `SanitisedError` by default.
```

Comment:
technically “JSDoc” but this isn’t really important ;)

Reply:
good spot 😁